pytest_plugins/external_logging.py

```diff
@@ -0,0 +1,27 @@
+from robottelo.config import settings
+
+
+def pytest_addoption(parser):
+    """Adds option for enabling external logging"""
+    help_text = '''
+        Flag for enabling promtail on the spawned hosts.
+        This is used for sending of the logs to the external Loki instance
+
+        Usage: --external_logging
+    '''
+    parser.addoption("--external-logging", action="store_true", default=False, help=help_text)
+
+
+def pytest_collection_modifyitems(config):
+    if not config.getoption('external_logging', False):
+        return
+    settings.server.deploy_arguments.promtail_enable = True
+    settings.capsule.deploy_arguments.promtail_enable = True
```

Comment:
Would suggest using settings.set instead of direct assigning value to override 
```suggestion
    settings.set('server.deploy_arguments.promtail_enable', True)
    settings.set('capsule.deploy_arguments.promtail_enable', True)
```

Reply:
done, thanks @Gauravtalreja1